ray-operator/test/e2e/counter.py

```diff
@@ -0,0 +1,24 @@
+import ray
```

Comment:
There are already a few versions of this code in the project. See for example https://github.com/ray-project/kuberay/pull/1460/files#diff-bc35e167b5138f35311d7d47ad10f7439d8e31404c47fc09baec1e74a87e95d1.
Can we have 1 version of it?

Reply:
What about updating the API server e2e tests to use that file from the operator e2e tests once this is merged? I'd rather have the API server module "depending" on the operator module, rather than the opposite.

Reply:
Between you and Laurentiu